src/components/container/FeaturedJobs/FeaturedJobs.scss

```diff
@@ -49,10 +49,37 @@
     }
   }
 
+  &__viewMoreLink__bottom {
+    display: none;
+  }
+
   &__cardContainer {
     display: flex;
     gap: 32px;
     justify-content: space-between;
     flex-wrap: wrap;
   }
+
+  @media screen and (max-width: 840px) {
```

Comment:
840 px seems a bit high as a threshold, but that would be something we could leave for the group to discuss. Will still approve because I guess this also triggers for tablets, which is actually quite nice.